changelogs/fragments/56900-route53-facts-check-mode.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
IMO this should be a bugfix, not a new feature. The module should have advertised that it supports check mode since the beginning.